docs/products/compute/compute-instances/guides/placement-groups/index.md

```diff
@@ -0,0 +1,213 @@
+
+## Availability
+
+<!--
+
+**CV: 03/2024** Availability at limited availability is only on two data centers - Washington D.C., and Miami. We'll be progressively opening up more regions as they're qualified. The request is to hide availability information in the docs, prior to actual GA.
+
+- UI: There's a mouse-over tooltip that reveals the available regions when creating a placement group. It'll be dynamically updated as new regions are made available.
+- API: The `GET /v4/regions:id` operation in the API spec lists `Placement Group` in the `capabilities` array, if PGs are supported for that region. This is the only op I see that shows support/non-support. The issue here is, you need to know the `id` for the target region, in order to run this operation. How would this work in an automated API workflow? Would you set up automation to periodically ping all regions, until it sees that `Placement Groups` is listed for a specific region? (In which case, the workflow could be manipulated to include the new region.)
+
+Because of the issue with the API, I'm including the Availability section in the docs. It's easy enough to update the docs on a weekly release cadence as we add new supported regions. (Do we anticipate qualifying regions quicker than once a week?)
+
+-->
+
+The Placement Groups service is available in select regions. Currently, this includes:
+
+- Miami, FL (us-mia)
+
+- Washington, DC (us-iad)
+
+{{< note >}}
+Placement Groups is in limited availability. Throughout this phase, we expect to increase the number of supported regions.
+{{< /note >}}
+
+## Affinity, enforcement, and compliance
+
+To distribute your compute instances in a placement group to meet your model, we use the industry-recognized affinity standard. You can pick from one of two types:
+
+- **Affinity**. Your compute instances are physically close together, possibly on the same host. Set this as your preferred container to help with performance.
+
+- **Anti-affinity**. Your compute instances are placed in separate fault domains, but they're still in the same region. Use this preferred container to better support a high-availability model.
+
+You also select how your affinity type should be enforced, when adding more compute instances to it in the future. This determines whether or not the placement group remains accessible – is “compliant”-- after you add a new compute instance:
+
+- **Strict (Best practice)**. You can't add more compute instances to your placement group if your preferred container lacks capacity or is unavailable. For example, let's assume you've set **Anti-affinity** as your affinity type. If you try to add a compute instance that's on the same host, or there's no capacity outside that host in the region, you get an error and can't add the compute instance. This helps you keep your placement group compliant, because you can only pick compute instances that fit your desired model.
```

Comment:
We haven't really defined what _container_ means in this context (unless I missed it).